### ISSUE TITLE
tutorial for dummies

### ISSUE THREAD

**Issue by camposdaltonton** [djmango/obsidian-transcription#5]
Is there any tutorial for people who don't know how to install manually. My experience so far with Obsidian is with the community plugins. I tried to copy the files to the plugins folder and managed to activate it, but when I give the command, it returns an error

**Comment by JuergenKaettnis** [comment]
Installing hasn't been the problem for me, but now I see I have to use docker to install s.th. called Whisper ASR. I don't about docker. I am on an Android tablet. Is the plugin suitable for Android devices?

**Comment by djmango** [comment]
Currently you need an instance of Whisper ASR to use the plugin - so you need either Python or Docker. Later this month I will release an update that will remove this dependency, so it will work plug-and-play on any device - stay tuned!

**Comment by thomas-neuburger** [comment]
Agree with the core suggestion. A step-by-step for *non*-programmers would be really helpful. I love this plugin, but it took many hours to learn everything I didn't know to get this thing running. 

A question: I managed to install the whisper command line program and get it running way before I got this plugin working. I expected the app to "see" and use the CLI. Apparently it doesn't, but uses Ahmed's implementation instead. Great idea, since Ahmed's "webservice", which runs locally after installation, is *very fast.*

OTOH, lots o' hours learning how to do that. (And there's an error in his own docs, a missing "sudo".)

Bottom line, *great* plugin. Would love better install and usage docs. 

(I suggested as a feature that Ahmed add "model" to the list of options at the localhost implementation. Are you going to add Ahmed's options to your plugin — for example, translate as well as transcribe, and various output formats like SRT? Just wondering.)

Thomas

**Comment by Porskrog** [comment]
Whisper works fine on my laptop where I have installed it using Python. But in Obsidian it fails. I think it is the URL localhost:9000 that is wrong. But I have no clue as I am not a tech-guy. Is there a more elaborate userguide on what to put in that field?

Thanks for all the good work on this.

**Comment by thomas-neuburger** [comment]
I got it to work, finally, but I don't now remember all the steps. I believe the secret sauce for Obsidian is to install the docker version of the app locally. That sets up localhost:9000 to work. 

I ended up following a chain of directions, linked from one page to another. Maybe start here:

https://github.com/ahmetoner/whisper-asr-webservice

and follow the Docker instructions. (I used instructions for CPU.) There are a bunch of steps, but I got it running — as I say, finally. Two notes:

- One of the commands needs a sudo that isn't shown.
- When you have to pick a keychain method (not the right word, but close), choose the less secure (older) one. That works out of the box, w/o more configuration.

HTH.

Thomas

**Comment by djmango** [comment]
https://www.youtube.com/watch?v=EyfhLGF3Fxg
